packages/web-components/src/horizontalbarchart/horizontalbarchart.ts

```diff
@@ -340,6 +373,9 @@ export class HorizontalBarChart extends FASTElement {
       .append('g')
       .each(createBars)
       .on('mouseover', function (event, d) {
+        if ((d as any).placeholder === true) {
```

Comment:
add explicit check that placeholder is defined and === true

Reply:
Okay